picard/formats/id3.py

```diff
@@ -494,6 +494,19 @@ def _get_tags(self, filename):
         except mutagen.id3.ID3NoHeaderError:
             return compatid3.CompatID3()
 
+    def _load_preserved_config(self, metadata):
+        # Adding additional config to metadata to prevent unecessary saving
+        metadata['~config:rating_steps'] = config.setting['rating_steps']
+        metadata['~config:rating_user_email'] = config.setting['rating_user_email']
```

Comment:
Here's a potential use case -
Lots of new files were added. 
We need a boolean anyways to store, if the file is being saved for the first time or not.

Since the overhead isn't much anyways to iterate through a couple of variables and initialise a dictionary. I don't think it should be much of a problem if we are saving config on a file basis.